lerna.json

```diff
@@ -1,9 +1,18 @@
 {
   "packages": [
```

Comment:
You can remove `packages` as it is duplicated with `workspaces` in package.json.